waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/model/ASTQueryMapping.java

```diff
@@ -56,6 +61,13 @@ public String transformOutboundDatabaseName(MetaStoreMapping metaStoreMapping, S
     return result.toString();
   }
 
+  boolean hasMarker(String query) {
```

Comment:
The function name `hasMarker` isn't super descriptive.  Can we change it to something like `hasIncompatibleViewMarker`, or `hasPrestoViewMarker`, or `hasNonHiveViewMarker`, or something like that?

Reply:
sure I like `hasNonHiveViewMarker`